woocommerce/woocommerce-gutenberg-products-block
# Add empty cart template to Cart i2

This PR closes #4793 
Know issues: because Cart i2 doen't have a skeleton, the Empty cart will be visible until the block is loaded. This will be fixed as part of #4964 

How to test this PR:

1. Go to a page and add the Cart i2 block
2. Make sure you can select the Empty cart view
3. Go to the front end with an empty cart
4. See that the empty cart is rendered
